Makefile

```diff
@@ -100,6 +104,10 @@ $(BUILD_DIR)/verify_boilerplate.py: $(BUILD_DIR)
 verify-go-mod: go-mod ## Verify the go modules
 	hack/tree-status
 
+.PHONY: verify-default-profiles
+verify-default-profiles: default-profiles ## Verify the generated default profiles
```

Comment:
Would it make sense to clear the profiles before regenerating them? That would guard against the case where the profiles were manually updated to be correct, but the generator was actually failing to run or not changing anything

Reply:
Hm, we indirectly clear them by regeneration all at once. If the generation outputs the same as the state of the repository, then we should be fine in any case. :shrug: